package.json

```diff
@@ -31,11 +31,14 @@
     "gulp": "gulp --gulpfile ./examples/gulpfile.js"
   },
   "dependencies": {
-    "gulp-util": "^3.0.8",
+    "ansi-colors": "^1.0.1",
+    "fancy-log": "^1.3.2",
     "lodash.template": "^4.4.0",
     "node-notifier": "^5.0.1",
     "node.extend": "^1.1.6",
-    "through2": "^2.0.3"
+    "plugin-error": "^0.1.2",
+    "through2": "^2.0.3",
+    "vinyl": "^2.1.0"
```

Comment:
`vinyl` should be in the `devDependencies`, it is only used for the tests.
Please update: it is important to avoid unnecessary dependencies.

Reply:
Thanks, the PR should be fine now.